### PR TITLE
Use big list of naughty strings to test string conversions

### PR DESCRIPTION
**Pull request by hoodmane**

### CODE REVIEW

File: src/tests/test_typeconversions.py [hunk 1]
```diff
@@ -64,6 +66,46 @@ def test_string_conversion2(selenium, s):
...
 
+def blns():
+    yield ""
+    import base64
+    import json
+    from urllib.request import urlopen
+
+    try:
+        with urlopen(
+            "https://raw.githubusercontent.com/minimaxir/big-list-of-naughty-strings/master/blns.base64.json"
```
Comment (ryanking13) [inline src/tests/test_typeconversions.py:77]: The list of strings are shorter than I think, so perhaps we can embed it instead of downloading it?

**Review by ryanking13 — APPROVED** [review]
Thanks, @hoodmane!

File: src/tests/test_typeconversions.py (hunk 1 above)
Comment (rth) [reply src/tests/test_typeconversions.py:77]: +1 possibly under `pyodide/src/tests/` but maybe gzip-compressed, to avoid appearing in grep.

**Review by rth — APPROVED** [review]

**Comment by hoodmane** [comment]
>  gzip-compressed

Well it's already base 64 encoded so I think it isn't a problem.

**Pull request merged**